data/transactions/logic/eval_test.go

```diff
@@ -5096,11 +5183,11 @@ func TestOpJSONRef(t *testing.T) {
 			json_ref JSONObject;
 			byte "key4";
 			json_ref JSONObject;
-			byte "key40"
+			byte "key40";
 			json_ref JSONString
```

Comment:
Up to you, but the semi-colons aren't needed when using multiline string

Reply:
Got it. It's used everywhere else in the test cases so I figured I'd be consistent.